app/controllers/passengers_controller.rb

```diff
@@ -61,7 +61,11 @@ def destroy
   private
 
   def passenger_params
-    params.require(:passenger).permit(:name, :address, :email, :phone,
+    #binding.pry
+    permitted_attrs = params.require(:passenger).permit(:name, :address, :email, :phone,
       :wheelchair, :active, :permanent, :expiration, :note)
+    expiration_date = permitted_attrs[:expiration]
+    permitted_attrs[:expiration] = Date.strptime(expiration_date, '%m/%d/%Y')
```

Comment:
:-1: There should definitely be a way to get the datepicker to send data in the format you need.